packages/react-components/components/BorderlessButton.v2.tsx

```diff
@@ -2,16 +2,15 @@ import Touchable, { Props as TouchableProps } from '@celo/react-components/compo
 import * as React from 'react'
 import { StyleProp, Text, TextStyle } from 'react-native'
 
-type Props = TouchableProps & {
+export type Props = Omit<TouchableProps, 'style'> & {
   style?: StyleProp<TextStyle>
-  testID?: string
 }
 
 // unstyled Touchable Text, good for making other Text Buttons such as TopBarButton
 export default function BoarderlessButton(props: Props) {
-  const { onPress, style, children, disabled, testID } = props
+  const { style, children, ...rest } = props
```

Comment:
Nit: I believe `passThroughProps` is a common convention for `rest` here

Reply:
Agreed 👍